pkgs/development/python-modules/pyfakewebcam/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, numpy, opencv4, ... }:
+
+buildPythonPackage rec {
+  pname = "pyfakewebcam";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "152nglscxmv7600i1i2gahny5z0bybnqgq3npak8npb0lsnwxn1a";
+  };
+
+  propagatedBuildInputs = [ numpy opencv4 ];
+
+  # No tests are available
+  doCheck = false;
+  pythonImportsCheck = [ "pyfakewebcam" ];
+
+  meta = with lib; {
+    description = "A library for writing RGB frames to a fake webcam device on Linux";
+    homepage = "https://github.com/jremmons/pyfakewebcam";
+    license = licenses.gpl3Plus;
```

The [License file](https://github.com/jremmons/pyfakewebcam/blob/master/LICENSE.txt) says LGPLv3.

```suggestion
    license = licenses.lgpl3Only;
```

---

Please inform the upstream about the licensing mismatch.

---

Thanks for catching this! Sorry about that

---

See https://github.com/jremmons/pyfakewebcam/pull/15